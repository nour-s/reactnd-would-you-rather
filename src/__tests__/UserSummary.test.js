import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import UserSummary from "../components/userSummary";

describe("UserSummary", () => {
	const location = { state: {} };
	const user = {
		name: "Tyler McGinnis",
		avatarURL: "../../images/tyler.jpg"
	};

	const setAuthedUser = () => {};

	it("Render UserSummary without crashing", () => {
		const renderer = new ShallowRenderer();
		renderer.render(<UserSummary user={user} />);
		const tree = renderer.getRenderOutput();
		expect(tree).toMatchSnapshot();
	});
});
