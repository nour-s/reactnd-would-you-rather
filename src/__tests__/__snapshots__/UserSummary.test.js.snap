// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSummary Render UserSummary without crashing 1`] = `
<div
  className="author-info"
>
  <img
    alt="avatar"
    src="../../images/tyler.jpg"
  />
  <span>
    Tyler McGinnis
  </span>
</div>
`;
